config/environment.js

```diff
@@ -49,9 +49,5 @@ module.exports = function (environment) {
     ENV.APP.autoboot = false;
   }
 
-  if (environment === 'production') {
-    // here you can enable a production-specific feature
-  }
-
```

Comment:
Why are we removing the production env setup?

Reply:
It is already added above and was a redundant block
https://github.com/Real-Dev-Squad/website-www/blob/679be1de26d3d0bfe27079af03cf8b437ae25563/config/environment.js#L22